core/src/main/res/values-v23/themes.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="Theme.Kevin.Base" parent="Theme.Kevin.Base.21">
+        <item name="android:statusBarColor">@android:color/transparent</item>
+        <item name="android:windowLightStatusBar">true</item>
+    </style>
+
+    <style name="Theme.Kevin.BottomSheet.Base" parent="Theme.Kevin.BottomSheet.Base.21">
+        <item name="android:statusBarColor">@android:color/transparent</item>
+        <item name="android:windowLightStatusBar">true</item>
+    </style>
```

Comment:
if we put spaces in the top, then let's place them in the bottom too before enclosing tag

Reply:
check other xml files too